rinja/src/lib.rs

```diff
@@ -174,9 +174,11 @@ impl<T: Template + ?Sized> Template for &T {
     const MIME_TYPE: &'static str = T::MIME_TYPE;
 }
 
-/// Object-safe wrapper trait around [`Template`] implementers
+/// [`dyn`-compatible] wrapper trait around [`Template`] implementers
 ///
-/// This trades reduced performance (mostly due to writing into `dyn Write`) for object safety.
+/// This trades reduced performance (mostly due to writing into `dyn Write`) for dyn-compatibility.
+///
+/// [`dyn`-compatible]: <https://doc.rust-lang.org/stable/reference/items/traits.html#object-safety>
```

Comment:
Waiting for <https://github.com/rust-lang/reference/pull/1512> to land, so the link fragment can be updated.

Reply:
Implemented in <https://github.com/rust-lang/reference/pull/1666>. The new version is not live, yet, but it will be soon enough.

Reply:
I don't know when it was releases, but the new link live now.